appcli/configuration_manager.py

```diff
@@ -315,15 +341,43 @@ def __seed_configuration_dir(self):
                 f"Seed file [{seed_app_configuration_file}] is not valid. Release is corrupt."
             )
 
+        # Create the configuration directory and copy in the app config file
         target_app_configuration_file = self.cli_context.get_app_configuration_file()
         logger.debug(
             "Copying app configuration file to [%s] ...", target_app_configuration_file
         )
-        target_app_configuration_file.parent.mkdir(parents=True, exist_ok=True)
+        os.makedirs(target_app_configuration_file.parent, exist_ok=True)
         shutil.copy2(seed_app_configuration_file, target_app_configuration_file)
 
-        template_overrides_dir = self.cli_context.get_template_overrides_dir()
-        template_overrides_dir.mkdir(parents=True, exist_ok=True)
+        # Create the configurable templates directory
+        logger.info("Copying configurable templates ...")
+        configurable_templates_dir = self.cli_context.get_configurable_templates_dir()
+        configurable_templates_dir.mkdir(parents=True, exist_ok=True)
+        seed_configurable_templates_dir = (
+            self.cli_configuration.configurable_templates_dir
+        )
+
+        # If a seed configurable templates directory is supplied
```

Comment:
Nested stuff like this is annoying. We should just:
```
if seed_configurable_templates_dir is None: 
  return

<the rest>
```

Also, the fine to leave the comments in, but they're not really adding much value (especially where you're error message explains why